src/envo/envo-idranges.owl

```diff
@@ -177,13 +177,21 @@ Datatype: idrange:18
     EquivalentTo:
         xsd:integer[> 3501100 , <= 3509999]
 
-Datatype: idrange:18
+Datatype: idrange:19
 
     Annotations:
         allocatedto: "Lauren Chan"
 
     EquivalentTo:
         xsd:integer[> 3510000 , <= 3519999]
 
+Datatype: idrange:20
+
+    Annotations:
+        allocatedto: "Raissa Meyer"
+
+    EquivalentTo:
+        xsd:integer[> 3520000 , <= 3529999]
```

Comment:
@raissameyer you could set that to 
 * ` xsd:integer[> 3519999 , < 3530000]` or 
 * ` xsd:integer[>= 3520000 , < 3530000]` or
 * ` xsd:integer[>= 3520000 , <= 3529999]` 

Please do to set a better precedent and save a number, I'll trust your sense of numeric aesthetics on which mode to assume.

Reply:
I'll choose `xsd:integer[>= 3520000 , <= 3529999]` because then the editor knows that all the numbers mentioned are actually used.